python/tests/test_pyvw.py

```diff
@@ -20,6 +20,9 @@ def test_learn_predict(self):
         assert init == 0
         self.model.learn(ex)
         assert self.model.predict(ex) > init
+        ex = ["| a", "| b"]
+        assert self.model.predict(ex) == [self.model.predict(ex[0]), self.model.predict(ex[1])]
+        self.model.learn(ex)
```

Comment:
This is incorrect, perhaps the issue was somewhat unclear. I believe allowing a list to be processed in this way because of multiline examples. If it is a list of strings then we should assume it is a multiline example